bin/build-app

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
-
+set -e
 python manage.py syncdb --noinput
-python manage.py migrate cabotapp --noinput
-python manage.py migrate djcelery --noinput
 python manage.py collectstatic --noinput
+python manage.py create_cabot_superuser
```

Comment:
create a superuser based on environment variables, eventually to allow cabot to run straight from a pushed docker image with no additional config

e.g. `docker run -e CABOT_SUPERUSER_USERNAME=root -e CABOT_SUPERUSER_PASSWORD=pass cabotapp:cabot`